indigo_app/templates/document/_comparison.html

```diff
@@ -0,0 +1,23 @@
+<div class="document-comparison-view d-none">
+  <header class="comparison-header">
+    <div class="title">
+      <h5>Comparison</h5>
+    </div>
+
+    <div class="buttons text-right">
+      <a href="#" class="btn btn-primary dismiss">Close</a>
+    </div>
+  </header>
+
+  <div class="comparison-main">
+    <div class="comparison-content">
+      <div class="document-sheet-container sheet-outer">
+        <div class="sheet-inner">
```

Comment:
If you put another div inside this and fill the `.akoma-ntoso` div with the comparison HTML, it'll be formatted correctly:

```html
<div class="sheet-inner">
  <div class="akoma-ntoso"></div>
</div>
```